sandcats/lib/dns.js

```diff
@@ -36,7 +36,7 @@ createDomainIfNeeded = function(mysqlQuery) {
   }
 }
 
-deleteRecordIfExists = function (wrappedQuery, domain, bareHost) {
+deleteRecordIfExists = function (wrappedQuery, domain, bareHost, type) {
```

Comment:
Now that I look at it, I guess I didn't need to add the `type` parameter here, since `A` records and ACME `TXT` records go on different hostnames anyway. Oh well.

Reply:
Actually this is outright wrong, I can't reuse this function. Fixing.